lua/entities/wtib_harvester_medium/init.lua

```diff
@@ -64,12 +63,19 @@ function ENT:Harvest()
 
 					WTib.ConsumeResource(self,"energy",Drain*1.2)
 					WTib.SupplyResource(self,"RawTiberium",Drain)
-					v:SetTiberiumAmount(v:GetTiberiumAmount()-Drain)
+					v:DrainTiberiumAmount(Drain)
+					
+					local ed = EffectData()
+						ed:SetEntity(self)
+						ed:SetStart(self:GetPos()+(self:GetUp()*self.Focus))
+						ed:SetOrigin(v:GetPos())
+						ed:SetScale(2)
+						ed:SetMagnitude(1)
+					util.Effect("wtib_debugtrace", ed)
```

Comment:
Even though it says its for debug, it though it looked cool, like the harvester was firing a laser at each crystal to harvest it.  It lets people know which crystals are being harvested.